include/sol/stack_core.hpp

```diff
@@ -1190,7 +1190,7 @@ namespace sol {
 		}
 
 		template <typename T>
-		decltype(auto) get_usertype(lua_State* L, int index = -lua_size_v<meta::unqualified_t<T>>) {
+		decltype(auto) get_usertype(lua_State* L, int index = -lua_size_v<meta::unqualified_t<T> >) {
```

Comment:
Maybe these changes should require some clang-format suppressions, otherwise it might be wiped in the next instance ?